### DISCUSSION TITLE
How to change server `api` folder name?

### DISCUSSION THREAD

**Discussion by meirroth** [nuxt/nuxt#27112] [Questions] [answered]
Hi fellow Nuxters,

How can I change the Nuxt/Nitro server API folder name from `/api` to `/_api` (or something)? Because our "main" backend is proxied at `/api`.

I couldn't find mention of this option in the Nuxt docs. 
Nitro shows the option https://nitro.unjs.io/config#apibaseurl but it didn't work when tried.

If anyone knows how to achieve this I'd greatly appreciate it.

https://stackblitz.com/edit/nuxt-api-config?file=server%2F_api%2Ftest.ts

**Comment by meirroth** [comment]
For now I found a workaround by moving the API event handlers under `server/routes/_api`. As far as I know they work the same 🤷‍♂️

Can pinia stores be accessed on the server?

**Comment by AloisSeckar** [comment, marked as answer]
Folder `/server/api` is there just for convenience, because it is most commonly used like this. If you need something else, `/server/routes` is there for you, as you already find out. I wouldn't even call it a "workaround" - it is a feature designed for use-cases like yours.

Regarding using Pinia server-side, from what I have found it is not possible directly. There is [one approach](https://stackoverflow.com/a/75801739/3204544) using plugin to populate the store from Nuxt API endpoint.

**Reply by meirroth** [reply]
@AloisSeckar Thank you for the fast response 😀
